### PR TITLE
Null Quaternion fix

### PR DESCRIPTION
**Pull request by benaadams**
Fix to issue pointed out in https://github.com/BabylonJS/Babylon.js/commit/bddbd42593058c414b8ea80722d067201266a602#commitcomment-17908278

### CODE REVIEW

File: src/Mesh/babylon.abstractMesh.ts [hunk 1]
```diff
@@ -763,7 +763,7 @@
             var yaw = -Math.atan2(dv.z, dv.x) - Math.PI / 2;
             var len = Math.sqrt(dv.x * dv.x + dv.z * dv.z);
             var pitch = Math.atan2(dv.y, len);
-            Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion);
+            this.rotationQuaternion = Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion || new Quaternion());
```
Comment (RaananW) [inline src/Mesh/babylon.abstractMesh.ts]: Quaternion.RotationYawPitchRollToRef doesn't return the quaternion. It only copies to the reference quaternion.
better would be to set the quaternion of the parent, if not set before. 
for example:
```javascript
this.rotationQuaternion = this.rotationQuaternion || new Quaternion() 
```

Comment (benaadams) [inline src/Mesh/babylon.abstractMesh.ts]: hmm it does return it; maybe behaviour is wrong tho?

Comment (RaananW) [inline src/Mesh/babylon.abstractMesh.ts]: Hey... that's totally new :) not even updated on my local repository yet.
This should work then, apologies.

Comment (RaananW) [inline src/Mesh/babylon.abstractMesh.ts]: Having said that - I do think that ToRef functions should not return anything. Might be a bit confusing. But it was merged, so it was also reviewed. 
I think the best solution would be to create the rotationQuaternion beforehand.

Comment (benaadams) [inline src/Mesh/babylon.abstractMesh.ts]: Fixed

**Pull request merged**